pom.xml

```diff
@@ -1887,6 +1888,17 @@
           @{jacocoArgLine}</argLine>
       </properties>
     </profile>
+    <profile>
```

Comment:
Do we need a profile for this ?
Specifying -DshadeSources=false would have the same effect.

But if you  prefer this way, it's fine, too.

Reply:
Yes actually just having a profile for a variable which can be passed from outside does not make sense. But I was thinking if we also add `maven.source.skip` to be true `here` and rename this profile to `skipSourceProcessing` or something.

Reply:
Hi @stoty Removed the profile as suggested by you.